jbpm/jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/ActivityTest.java

```diff
@@ -966,6 +1002,29 @@ public void testServiceTask() throws Exception {
         assertProcessInstanceFinished(processInstance, ksession);
         assertEquals("Hello john!", processInstance.getVariable("s"));
     }
+    
+    @Test
+    public void testServiceTaskWithAccessoWorkItemInfo() throws Exception {
```

Comment:
```suggestion
    public void testServiceTaskWithAccessToWorkItemInfo() throws Exception {
```
is that what you mean?

Reply:
yes, indeed that was what I meant, will update

Reply:
fixed